cmd/gcp-controller-manager/app/node_annotater_test.go

```diff
@@ -182,6 +184,9 @@ func TestExtrackKubeLabels(t *testing.T) {
 			if got, want := out, c.out; !reflect.DeepEqual(got, want) {
 				t.Errorf("unexpected labels\n\tgot:\t%v\n\twant:\t%v", got, want)
 			}
+			if c.expectNoMetadataErr && err != errNoMetadata {
+				t.Errorf("unexpected errNoMetadata")
+			}
 			if got, want := (err != nil), c.expectErr; got != want {
```

Comment:
if got, want := (err != nil), c.expectErr || c.expectNoMetadataErr; got != want {

Reply:
I set:

c.expectErr = c.expectErr || c.expectNoMetadataErr

above but I'll move it here.